src/styles/generic/_buttons.scss

```diff
@@ -7,9 +7,6 @@
 // =============================================================================
 
 button {
+  margin: 16px 16px 0 0;
```

Comment:
I think this may add unnecessary top margin to all of our buttons, the measure page is a good example:

![image](https://user-images.githubusercontent.com/1066253/68518200-bae53480-023f-11ea-9541-768538997755.png)

For what it's worth, most folks don't use buttons in their article. So I'm wondering if we should just ask authors to put their buttons into a container that adds the necessary margin at smaller screen sizes? @argyleink any suggestions?

Reply:
I think the only sane way to tackle this is to let the parent container decide margins of the buttons inside.